.github/workflows/ci.yml

```diff
@@ -3,17 +3,17 @@ name: Test and build
 on: [push, pull_request, release]
 
 jobs:
-  qa:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-python@v4
-        with:
-          python-version: "3.9"
-      - uses: pre-commit/action@v3.0.0
+  # qa:
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@v4
+  #     - uses: actions/setup-python@v5
+  #       with:
+  #         python-version: "3.9"
+  #     - uses: pre-commit/action@v3.0.0
```

Comment:
Is QA broken just for ubuntu? (I guess so otherwise you wouldn't be able to make commits)

Reply:
It's not broken,  but I've turned on pre-commit.ci, so was removing the duplication.